packages/client/wallets/walletconnect/src/hooks/useWalletConnectRequests.tsx

```diff
@@ -96,7 +102,7 @@ export function WalletConnectRequestsContextProvider({ children }: { children: R
                 "[WalletConnectRequestsContextProvider.acceptRequest()] failed to respond to session request",
                 e
             );
-            // TODO: surface error to user
+            toast.error(`Failed to respond to ${method} request`);
```

Comment:
This doens't seem super user friendly, can you think of something that my dad could more or less understand or at least not get scared about

Reply:
i have a function that parses the method names better in a future pr, ill use that one once its in